test_scripts/Polices/External_UCS_Informing_HMI/002_ATF_P_GetListofPermissions_with_appID_all_ecs_lower_bound_EntitySatus_ON.lua

```diff
@@ -50,9 +50,26 @@ end
 function Test:Precondition_PTU_and_OnAppPermissionConsent_AllParams_Lower()
   local ptu_file_path = "files/jsons/Policies/Related_HMI_API/"
   local ptu_file = "OnAppPermissionConsent_ptu.json"
+  local time_onAppPermissionChanged = 0
 
   testCasesForPolicyTable:flow_SUCCEESS_EXTERNAL_PROPRIETARY(self, nil, nil, nil, ptu_file_path, nil, ptu_file)
 
+  EXPECT_NOTIFICATION("OnPermissionsChange")
+  :Times(2)
```

Comment:
@Itileda, based on requirements expect to have 2 notifications: 
First notification is received after successful PTU where application gets new functions groups. See "[OnPermissionsChange] send after app's permissions change by Policy Table Update "
Second notification is received after user allows definite permissions. See "[Policies]: User-consent "YES""